src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -82,4 +68,12 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+
+    default String getPrintableString(Printable... printables) {
+        StringJoiner sj = new StringJoiner(" ");
```

Comment:
StringBuilder or StringJoiner can both be used. Good that one of the mentioned is used here.